6.3.22/script.js

```diff
@@ -0,0 +1,33 @@
+/*------------------------------------------------------*/
+const dishes = ['Spaghetti', 'Lasagne', 'Beans on toast', 'Porridge', 'Tomato soup',
+'Salmon'];
+dishes.food = "I am hungry";
```

Comment:
//you would use dished.food if you wanted to insert the food key with "I am hungry" value into an object